specification.md

```diff
@@ -23,9 +23,10 @@
     + [Using multiple data filters](#using-multiple-data-filters)
     + [Data Merging](#data-merging)
   * [Workflow Functions](#workflow-functions)
-    + [Using Functions for RESTful Service Invocations](#using-functions-for-restful-service-invocations)
+    + [Using Functions for OpenAPI-based RESTFul Service Invocations](#using-functions-for-openapi-based-restful-service-invocations)
+    + [Using Functions for HTTP Service Invocations](#using-functions-for-http-service-invocations)
     + [Using Functions for Async API Service Invocations](#using-functions-for-async-api-service-invocations)
-    + [Using Functions for RPC Service Invocations](#using-functions-for-rpc-service-invocations)
+    + [Using Functions for gRPC Service Invocations](#using-functions-for-rpc-service-invocations)
```

Comment:
please rename the header where this links to: #using-functions-for-rpc-service-invocations